csrc/multidevice/executor.h

```diff
@@ -69,18 +70,7 @@ namespace nvfuser {
      device scheduling.
 */
 
-struct MultiDeviceExecutorParams {
-  // Experimental: whether to use FusionExecutorCache rather than
-  // FusionExecutor.
-  bool use_fusion_executor_cache = false;
-  // Experimental: whether to apply auto-scheduling in FusionExecutorCache if
-  // use_fusion_executor_cache=true. WAR: temporary hack mainly use for
-  // development
-  bool skip_auto_scheduling = false;
-  // Experimental: whether to cache fusion executor. WAR: avoid recompilation
-  // but implicitely assumes that the input shape don't change over iterations
-  bool cache_fusion_executor = false;
-};
+using MultiDeviceExecutorParams = hir::HostIrExecutorParams;
```

Comment:
I'd just replace MultiDeviceExecutorParams. I don't see the value of creating an alias.

Reply:
agreed